profile/README.md

```diff
@@ -26,16 +26,16 @@ You can also browse by Awesome Lists created by the community.
 
 3. Now either:
 
-     - **Start a new community extension** by [opening an issue](https://github.com/Camunda-Community-Hub/community/issues/new/choose) using the `New Community Extension Proposal` template and following [this documentation.](https://docs.github.com/en/github/administering-a-repository/transferring-a-repository#transferring-a-repository-owned-by-your-organization)
-     - [**Transfer your existing repository**](community/transferring-extensions.md) into the Camunda Community Hub.
-     - **Browse our existing projects** and issues and contribute.
+     - **Start a new community extension** by [opening an issue](https://github.com/Camunda-Community-Hub/community/issues/new/choose) using the `New Community Extension Proposal` template.
+     - [**Transfer your existing repository**](https://github.com/camunda-community-hub/community/blob/main/transferring-extensions.md) into the Camunda Community Hub.
+     - [**Browse our existing projects**](https://github.com/orgs/camunda-community-hub/repositories) and issues and contribute.
 
 We’ve put together some commonly asked [questions and answers here.](https://github.com/camunda-community-hub/community) 
 
 ### Contributor resources & documentation
 * [Contributor guide](https://github.com/camunda-community-hub/community/blob/main/CONTRIBUTING.MD)
 * [Issue triage and labeling](https://github.com/camunda-community-hub/community/blob/main/issue-triage.md)
-* [Maintainer & reviewer expectations]([#maintainer-reviewer-expectations.md](https://github.com/camunda-community-hub/community/blob/main/maintainer-reviewer-expectations.md))
+* [Maintainer & reviewer expectations](https://github.com/camunda-community-hub/community/blob/main/maintainer-reviewer-expectations.md)
```

Comment:
Same here. Is it not building correctly if you use `../../community/maintainer-reviewer-expectations.md`?

Reply:
Same for this :)